jicoco/pom.xml

```diff
@@ -32,8 +32,8 @@
     <description>Jitsi Common Components</description>
 
     <properties>
-        <jetty.version>9.4.44.v20210927</jetty.version>
-        <jersey.version>2.35</jersey.version>
+        <jetty.version>9.4.46.v20220331</jetty.version>
+        <jersey.version>3.0.4</jersey.version>
```

Comment:
Hey @ibauersachs why is this change needed at the moment? Can we leave it for later as this breaks the bridge:

```
JVB 2022-05-16 11:56:15.470 WARNING: [1] org.glassfish.jersey.server.wadl.WadlFeature.configure: JAXBContext implementation could not be found. WADL feature is disabled.
Exception in thread "main" java.lang.NoClassDefFoundError: jakarta/ws/rs/core/Response
```

Reply:
Because 2.35 has transitive dependencies with high severity CVEs (in Jackson Databind). I somewhat expected this to break stuff, but didn't have enough time to investigate what other changes are necessary. Probably updating Jetty to a modern version would solve it.

Feel free to revert it, but it'll bring the CVEs back unless you exclude them all and manually add the patched versions as direct dependencies...